package.json

```diff
@@ -10,19 +10,20 @@
     "url": "git@github.com:BBVAEngineering/ember-cli-htmlbars-minifier.git"
   },
   "scripts": {
-    "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint --max-warnings 0 ./*.js lib tests",
     "check": "node_modules/.bin/check-dependencies",
-    "pretest": "npm run check && npm run lint:js && npm run clean-coverage",
-    "test": "npm run test:node",
-    "test:node": "nyc --reporter=html --reporter=json --reporter=json-summary --reporter=lcov --report-dir coverage --exclude tests npm run mocha",
-    "mocha": "node_modules/.bin/mocha --recursive tests",
     "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 95 --functions 100 --branches 85 --lines 95",
     "clean-coverage": "rm -rf coverage",
     "commit": "git-cz",
+    "lint:js": "eslint --max-warnings 0 ./*.js config tests lib node-tests",
+    "mocha": "node_modules/.bin/mocha node-tests --recursive",
     "posttest": "npm run report-coverage && npm run check-coverage",
+    "pretest": "npm run check && npm run lint:js && npm run clean-coverage",
     "report-coverage": "istanbul report --include=coverage/coverage-final.json text",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "test": "npm run test:node && npm run test:ember",
+    "test:all": "ember try:each",
+    "test:ember": "ember test",
```

Comment:
You missed `COVERAGE=true` flag. You need to add `coverage.js` file as well.

Reply:
I intentionally removed it, because we don't have testeable code of Ember in this addon (only the dummy app, excluded of this purpose)